src/Init/Data/String/Basic.lean

```diff
@@ -455,6 +455,10 @@ instance : Inhabited String := ⟨""⟩
 
 instance : Append String := ⟨String.append⟩
 
+@[simp]
```

Comment:
Very minor, but unless there is a line length issue you can just put `@[simp]` on the same line, e.g. `@[simp] theorem ...`.